A number of minor (last-minute) changes

WilliamMass opened the pull request:
Turns "Getting Started with the Portal API" into the documentation landing page

Clarifies some points on the getting started page.

Fixes redundancy on freewaydata endpoint page.

WilliamMass committed:
Update navigation.yml

Routes 'Documents' navbar link to /documents/gettingstarted/ instead of to the documents folder

WilliamMass committed:
Update freewaydata.md

Remove extraneous explanation of start_date and end_date parameters.

WilliamMass committed:
Update _navigation.scss

Change nav button display to 'none', as a test, without messing with any other pagination settings.

WilliamMass committed:
Update _navigation.scss

Revert last change.

WilliamMass committed:
Update gettingstarted.md

Edited for clarity.

Formatting and Standardization.

leetam commented:
@WilliamMass after some consideration, I don't think the "Getting Started with the Portal API" should be the documentation landing page. The documentation landing page should provide some general description of what has been documented. Please revert the documentation landing page to its original and I'll follow-up with a suggestion for the documentation landing page in an email later this week. Once the the landing page has reverted please submit a new pull request. Thanks!

WilliamMass committed:
Update navigation.yml

Revert change to documentation routing in advance of creating a new landing page.

(Landing page is no longer getting started with the API)

WilliamMass commented:
Closing this pull request in order to re-submit with suggested changes.
